### PR TITLE
Ft add favicon

### PR DESCRIPTION
**Pull request by Philimuhire**
**PR Description**
This PR adds a favicon next to the title of the webpage in the browser's tab.

**Description of tasks that were expected to be completed**
Add a favicon

**How has this been tested?**
. Clone the repo
. Checkout branch ft-add-favicon
. Run the project with npm run dev

**Track PR (issue number & link)**
Issue link : https://github.com/atlp-rwanda/atlp-devpulse-fn/issues/115

**Screenshoots**

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "start": "echo \"Error: no test specified\" && exit 1",
-    "dev": "webpack server",
+    "dev": "webpack serve --port 3005",
```
Comment (mugishaj092) [inline package.json]: I believe we don't need to specify the port here, as it has already been set in the webpack.config.js file.

File: public/index.html
Comment (mugishaj092) [inline public/index.html:1]: I also think this file might not be necessary, as we are already using the index.html located in the root directory.

**Review by mugishaj092 — CHANGES_REQUESTED** [review]
Fix codeclimate and rebase also logo is not visible 
image

File: public/index.html
Comment (Philimuhire) [reply public/index.html:1]: @mugishaj092 This file was already there before I make changes. Could it be better to remove it?

File: package.json [hunk 2]
```diff
@@ -103,8 +105,9 @@
     "jwt-decode": "^3.1.2",
     "mini-css-extract-plugin": "^2.6.1",
     "moment": "^2.29.4",
-    "next-link": "^2.0.0",
```
Comment (mugishaj092) [inline package.json:106]: We have issue on this package

**Review by mugishaj092 — CHANGES_REQUESTED** [review]

**Review by robsdagreat — APPROVED** [review]

**Review by hbapte — APPROVED** [review]
well done

**Comment by Prince-Kid** [comment]
Well Done

**Review by Prince-Kid — APPROVED** [review]
looking good

**Review by mugishaj092 — CHANGES_REQUESTED** [review]
looks good so far but this build file is not nessary

**Review by mugishaj092 — APPROVED** [review]

**Review by mugishaj092 — APPROVED** [review]

**Pull request merged**